README.md

```diff
@@ -58,84 +58,131 @@ Here is an overview of all fields that can be present in plugin meta YAML files
-| id | plugin ID | yes |
+| publisher | publisher name | yes |
+| deprecate.migrateTo | which plugin should be used instead when plugin is deprecated | no |
+| deprecate.autoMigrate | if deprecated plugin should be automatically replaced by another plugin specified in deprecate.migrateTo | no |
+| name | plugin name | yes |
 | version | plugin version | yes |
 | type | plugin type | yes|
-| name | plugin name | yes|
+| displayName | plugin name | yes|
 | title| plugin title | yes |
 | description| plugin description | yes |
 | icon| URL to plugin icon (must be in SVG format) | yes |
 | url| an URL to the plugin source | yes |
 | publisher| name of plugin publisher | yes |
 | repository | URL to repository of the plugin | yes |
-| category| plugin category | yes<sup>1</sup> | 
+| category| plugin category | yes<sup>1</sup> |
 | firstPublicationDate | date of publishing the plugin (in ISO 8601) | no<sup>2</sup> |
 | latestUpdateDate | date of latest plugin update (in ISO 8601) | no<sup>3</sup> |
 | preview | a URL to devfile, to preview this plugin | no |
-| tags | a list of tags, related to this plugin | no| 
+| tags | a list of tags, related to this plugin | no|
 | mediaImage | links for images showcasing the plugin | no |
 | mediaVideo | links for video showcasing the plugin| no |
 | attributes | a map of special attributes, can be used for instance, imported plugins from VS Code | no |
 
 1 - Category must be equal to one of the following: "Editor", "Debugger", "Formatter", "Language", "Linter", "Snippet", "Theme", "Other"
 
 2 - firstPublicationDate is not required to be present in YAML, as if not present, it will be generated during Plugin Registry dockerimage build
- 
-3 - latestUpdateDate is not required to be present in YAML, as it will be generated during Plugin Registry dockerimage build 
 
-At the moment, some of these fields (that are related to plugin viewer) are validated during the Plugin Registry dockerimage build. 
+3 - latestUpdateDate is not required to be present in YAML, as it will be generated during Plugin Registry dockerimage build
+
+At the moment, some of these fields (that are related to plugin viewer) are validated during the Plugin Registry dockerimage build.
 
 ## Get index list of all plugins
 Example:
 ```
 curl  "http://localhost:8080/plugins/index.json"
 ```
+or
+```
+curl  "http://localhost:8080/plugins/"
+```
 Response:
 ```javascript
 [
   {
-    "id": "org.eclipse.che.editor.theia",
+    "name": "theia",
     "version": "1.0.0",
+    "publisher": "che-incubator",
+    "id": "che-incubator/theia/1.0.0",
+    "type": "Che Editor",
+    "displayName": "theia-ide",
+    "description": "Eclipse Theia",
+    "links": {
+      "self": "/plugins/che-incubator/theia/1.0.0"
+    }
+  },
+  {
+    "name": "theia",
+    "version": "next",
+    "publisher": "che-incubator",
+    "id": "che-incubator/theia/next",
     "type": "Che Editor",
-    "name": "theia-ide",
+    "displayName": "theia-ide",
```

Comment:
`Eclipse Theia` would be better as an example of `displayName` field

Reply:
done